server/.vs/server/config/applicationhost.config

```diff
@@ -0,0 +1,1014 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
`.vs` directory should be added to .gitignore. Add it and remove this file from pull request. Same goes to every other file from this directory